pages/settings/wallets/ConnectWallet.tsx

```diff
@@ -171,55 +175,60 @@ export function ConnectWallet() {
           </TouchableOpacity>
         )}
       />
-      {!supportsCreateConnection && <NotSupportedView />}
-      {supportsCreateConnection && (
-        <>
-          <View className="flex-1 justify-center items-center gap-8 p-6">
-            <PlugView
-              connectionCreated={connectionCreated}
-              name={name}
-              icon={icon}
-            />
-            {connectionCreated && (
-              <ConnectedView
-                returnTo={returnTo}
-                redirectCountdown={redirectCountdown}
-              />
-            )}
-            {!connectionCreated && (
-              <ConnectView
-                name={name}
-                requestMethods={requestMethods}
-                notificationTypes={notificationTypes}
-                isolated={isolated}
-                budgetRenewal={budgetRenewal}
-                satsAmount={satsAmount}
-                expiresAt={expiresAt}
-                returnTo={returnTo}
-                metadata={metadata}
-              />
-            )}
+      <View className="flex-1 justify-center items-center gap-8 p-6">
+        <PlugView
+          connectionCreated={connectionCreated}
+          name={name}
+          icon={icon}
+        />
+        {!supportsCreateConnection ? (
+          <View className="flex-1 justify-center items-center p-8">
+            <Text className="text-center">
+              This wallet connection does not support one tap connections.
+            </Text>
+            <Text className="text-center mt-4">
+              {hasCreateConnectionWallet
+                ? "Please switch your wallet and try again."
+                : `Please re-connect Alby Go from Alby Hub using the "One Tap Connections" button.`}
```

Comment:
Not sure if this copy is okay. I thought it's good to track if there's any wallet with `create_connection` and show the message accordingly.

Reply:
Yeah, I agree. It should be good for now until some other wallet service also supports it